### PR TITLE
Remove objectstore credentials

### LINKED ISSUE #17659: nextcloud.occ config:list prints object storage credentials
The `nextcloud.occ config:list` outputs the object storage credentials in plaintext 

The [Issue Debugging Script](https://github.com/nextcloud/nextcloud-snap/wiki/Issue-Debugging-Scripts) relies on this command and this can lead to posting access credentials publicly.

### Steps to reproduce
1. Setup nextcloud with object storage as the primary storage medium
2. `nextcloud.occ config:list`

### Expected behaviour
Credentials get removed/replaced with `***REMOVED SENSITIVE VALUE***` as it is done for the `passwordsalt` and `secret` field (amoung others). 

### Actual behaviour
Key and Secret are shown.

### Output
```
        "objectstore": {
            "class": "OC\\Files\\ObjectStore\\S3",
            "arguments": {
                "bucket": "mydata.nextcloud.myDomain.tld",
                "region": "eu-central-1",
                "autocreate": true,
                "key": "<Key was included in plaintext>",
                "secret": "<Secret was included in plaintext>",
                "use_ssl": true
            }
        },
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
```

Issue first opend for the debugging script in nextcloud-snap, however it should fit better here. 
[https://github.com/nextcloud/nextcloud-snap/issues/1150](https://github.com/nextcloud/nextcloud-snap/issues/1150)

### PR DESCRIPTION
**Pull request by kesselb**
Close #17659

S3: key and secret
Swift v2: also remove the username
Swift v3: user.name and user.password

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

File: lib/private/SystemConfig.php [hunk 1]
```diff
@@ -64,13 +64,24 @@ class SystemConfig {
 		],
 		'objectstore' => [
 			'arguments' => [
-				'password' => true,
+				// @Todo: Is this still valid?
```
Comment (rullzer) [inline lib/private/SystemConfig.php]: yeah keep it

Comment (kesselb) [reply lib/private/SystemConfig.php]: Is this a change request? :thinking: Todo refers to `options => ['credentials => [...]]`. `password` is still there just moved it below to Swift v2. I could not find any usage for the options syntax.

Comment (rullzer) [reply lib/private/SystemConfig.php]: yeah it is ancient swift code but could still be in use

File: lib/private/SystemConfig.php [hunk 2]
```diff
@@ -64,13 +64,24 @@ class SystemConfig {
 		],
 		'objectstore' => [
 			'arguments' => [
-				'password' => true,
```
Comment (nickvergessen) [inline lib/private/SystemConfig.php]: Let's keep this in case of a verify old/wrong config?

**Review by nickvergessen — APPROVED** [review]
Fine in general, but I would keep the password thingy just to be sure

File: lib/private/SystemConfig.php (hunk 2 above)
Comment (kesselb) [reply lib/private/SystemConfig.php]: Yes. I moved it down to Swift v2 and added username.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**